scripts/enhance-requires

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+version_deps () {
+	while read dep
+	do
+		if [[ "${dep}" =~ ^[^\(]*\(\) ]] && rpm -q --whatprovides "${dep}" &> /dev/null
+		then
+			printf "($dep with %s)\n" "$(rpm -q --whatprovides "${dep}" --qf '%{NAME} >= %{VERSION}')"
```

Comment:
Why not make this a separate dependency, i.e. keep the original dep as it was, and just add a separate version dep?

I think `%{?_isa}` should be used, i.e. the dependency should specify the arch, so that a package for another arch is not used to satisfy the dep inadvertently.

Reply:
> keep the original dep as it was, and just add a separate version dep?

My thinking was that "Requires: libnghttp2.so.14()(64bit)", "Requires: libnghttp2 >= 1.51.0" could theoretically be satisfied by two separate packages (possibly including a package of the correct arch that provided an outdated version of the so, and an up to date version of the package of the wrong arch, if they did not require matching versions of each other by one means or another).  

Requiring "(libnghttp2.so.14()(64bit) with libnghttp2 >= 1.51.0)" should be as unambiguous as the current implementation, with regard to the arch, while adding version info.